policy/modules/system/systemd.fc

```diff
@@ -104,3 +104,13 @@ HOME_DIR/\.config/systemd/user(/.*)?		gen_context(system_u:object_r:systemd_unit
 /var/run/systemd/units(/.*)?		gen_context(system_u:object_r:systemd_unit_file_t,s0)
 
 /var/run/initramfs(/.*)?	<<none>>
+
+# systemd-homed
+/usr/lib/systemd/system/systemd-homed-activate\.service	--	gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+/usr/lib/systemd/system/systemd-homed\.service		--	gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+/usr/lib/systemd/systemd-homed				--	gen_context(system_u:object_r:systemd_homed_exec_t,s0)
+/usr/lib/systemd/systemd-homework			--	gen_context(system_u:object_r:systemd_homed_exec_t,s0)
+/var/lib/systemd/home(/.*)?					gen_context(system_u:object_r:systemd_homed_var_lib_t,s0)
+/var/run/systemd/home(/.*)?					gen_context(system_u:object_r:systemd_homed_runtime_t,s0)
+/var/run/systemd/user-home-mount(/.*)?				gen_context(system_u:object_r:user_home_dir_t,s0)
```

Comment:
I suspect this is wrong.Unfortunately I cannot elaborate on why I think it is wrong. I wrote policy for homed/userdbd back when it was introduces but unfortunately I deleted that repository and I just seem to recall not doing that.  What if you would just label that systemd_home_runtime_t as well?